include/psxsdk/libgpu.h

```diff
@@ -220,6 +220,12 @@ extern u_short LoadTPage(PixPattern* pix,  // Pointer to texture pattern start a
 
 // clang-format on
 
+/**
+ * Calculates and returns the texture CLUT ID.
+ * The CLUT address is limited to multiples of 16 in the x direction
+ */
+u_short GetClut(int x, int y // Frame buffer address of CLUT
```

Comment:
Remember to not put comments that are too long on the same line of code to avoid weird auto formatting

Reply:
Looks good to me